packages/python-sdk/e2b/sandbox_async/commands/command.py

```diff
@@ -201,16 +206,28 @@ async def run(
         cwd: Optional[str] = None,
         on_stdout: Optional[OutputHandler[Stdout]] = None,
         on_stderr: Optional[OutputHandler[Stderr]] = None,
+        stdin: Optional[bool] = None,
```

Comment:
Not 100% sure about this signiture? Should this be `False`? But that we can't check if the usage is incorrect